src/app/middlewares/errorHandler.ts

```diff
@@ -1,4 +1,4 @@
-import log from "@src/services/Logger";
+import log from "@src/common/Logger";
```

Comment:
```suggestion
import * as log from "@src/common/Logger";
```

export default문 대신 export 문으로 사용하려면 다음과 같이 쓸 수 있지 않을까요..? 대신 내부는 class가 아닌 단순 export 함수들로 작성되어야 할 것 같네요.. 내부함수들을 클로저로 참조하는 방식으로요